client/src/app/site/site.component.ts

```diff
@@ -45,6 +48,20 @@ export class SiteComponent implements OnInit {
         socket.next(val => {
             console.log('socket.next: ', val);
         });
+
+        //get a translation via code: use the translation service
+        this.translate.get('Motions').subscribe((res: string) => {
```

Comment:
@tsiegleauq Is there a possibility to get the result synchronously? We need the translations very often in the code, and doing everythin translationrelated asynchronly will result in much more complex code.